es/refresher-linear-algebra.md

```diff
@@ -216,7 +216,7 @@
 
 **37. For x∈V, the most commonly used norms are summed up in the table below:**
 
-&#10230; Para x∈V, los módulos o normas más comúnmente usadas están descritas en la tabla de abajo:
+&#10230; Sea x∈V, las normas comunmente utilizadas se resumen en la siguiente tabla:
```

Comment:
* Add the accent to "_comúnmente_". 
* I also agree with gustavo. Let's use "sea" only when the text to traduce is "given", although it sounds less technical.